MusicApp/app/src/main/java/com/hungngo/musicapp/service/MyMusicService.kt

```diff
@@ -0,0 +1,221 @@
+package com.hungngo.musicapp.service
+
+import android.app.*
+import android.content.Context
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.media.MediaPlayer
+import android.net.Uri
+import android.os.Build
+import android.os.Bundle
+import android.os.Handler
+import android.os.IBinder
+import android.support.v4.media.session.MediaSessionCompat
+import android.widget.Toast
+import androidx.core.app.NotificationCompat
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import com.hungngo.musicapp.Constance
+import com.hungngo.musicapp.MyReceiver
+import com.hungngo.musicapp.R
+import com.hungngo.musicapp.data.model.Song
+import kotlin.random.Random
+
+class MyMusicService : Service() {
+    private var mediaPlayer: MediaPlayer? = null
+    private var isPlaying: Boolean = false
+    private var position = 0
+    private lateinit var listSongs: MutableList<Song>
```

Comment:
Hạn chế dùng late init nha , ko là dễ null 
```suggestion
    private var listSongs = mutableListOf<Song>()
```